articles/atom-vs-visual-studio-code-functionality-and-features-comparison/index.md

```diff
@@ -0,0 +1,178 @@
+| Auto-completion | Yes | Yes |
+| Inbuilt Version Control | Yes | Yes |
+| Multiple selection editing | Yes | Yes |
+| Price | Free | Free |
+| Find and replace | Yes | Yes |
+
+### Atom vs. VS Code: Features and Functionality Comparison
+
+1. Configuration
+
+With loads of extensions and customization comes many configurations. The configuration of applications is fundamental. It determines the overall user experience with the editor.
+
+Configuration in Visual Studio Code involves a JSON file recently replaced by Graphical User Interface (GUI). It is simpler to use GUI, and it does its work well. The level of abstraction is maintained while using GUI over JSON.
+
+In Atom, you have GUI everywhere to edit instead of having a singular JSON file to edit. Atom users report there are sluggishness and other performance issues during the configuration process. This happens when tons of extensions are installed. The solution to these issues is doing away with extensions not in use and maintaining a relatively small number of installed extensions.
+
+2. Extensibility and customization
+
+Atom and Visual Studio Code editors are extensible and customizable with third-party add-on packages. Both editors allow you to search extensions, install, and manage the extensions directly inside the program. The one thing that differs between the two editors is:
+
+In Visual Studio Code, themes are regarded as an extension like any other. In Atom, themes are a different category of extension and are installed and managed differently.
+
+3. Plugins and integration
+
+Extensibility is where Atom and Visual Studio Code editors differ hugely. In the Visual Studio Code, plugins add more features and functionality to the program. The features that can be added include language support, themes, commands, Git integration, Markdown support, debuggers, and more.
+
+Atom gives more power and functionality to plugins. Atom has tons of built-in and third-party plugins. This is where the editor functionality comes from, thus being a hackable editor.
+
+4. Git Integration
+
+Atom is a product of GitHub, and git integration is an in-built feature. On the other hand, Visual Studio Code has many GitHub related extensions.
+
+5. Auto-completion
+
+The autocomplete tool lets you view and insert possible completions in the editor using the Enter button. The autocomplete system works by looking through the currently open file for strings that match whatever you type. Both Atom and Visual Studio Code editors have the autocomplete feature.
+
+In the Atom code editor, the autocomplete functionality is implemented in [the autocomplete-plus package](https://github.com/atom/autocomplete-plus). VS Code comes integrated with tools for auto-completion, thus making the code you write more convenient. Tools for auto-completion in VS Code include: [Emmet](https://code.visualstudio.com/docs/editor/emmet) and [IntelliSense.](https://code.visualstudio.com/docs/editor/intellisense)
```

Comment:
Please illustrate how Atom's AutoComplete Plus is different from IntelliSense and Emmet.

Please also illustrate how powerful Emmet is for web development by a small example.